### ISSUE TITLE
Marker is not being shown if it is added after map was added to its container

### ISSUE THREAD

**Issue by paodb** [FlowingCode/GoogleMapsAddon#59]
This comes from directory:

_"I have to add the GoogleMap object to its container last. Any other place in the code no GoogleMapMarker are shown at first show of the map (I had to remove and add them again)."_

It should be possible to add a marker after the map is added to the container. 

Example to reproduce this:

```java
      GoogleMap gmaps = new GoogleMap(apiKey, null, null);
      gmaps.setMapType(MapType.ROADMAP);
      gmaps.setCenter(new LatLon(-31.636036, -60.7055271));
      gmaps.setSizeFull();     
      add(gmaps);
      gmaps.addMarker(
          new GoogleMapMarker("You are here!", gmaps.getCenter(), false, Markers.GREEN));
```

**Comment by paodb** [comment]
After some testing, I discover issue can be fixed in web-component so I've created [this new ticket](https://github.com/FlowingCode/google-map/issues/13) in order to work on the fix. Once the release of the fix in the web-component is done, I will update the add-on with the new version.

**Comment by paodb** [comment]
Fixed by https://github.com/FlowingCode/google-map/issues/13.

**Comment by paodb** [comment]
Fix is out with new release 1.6.0.
